<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_ui_message">
    <sys_ui_message action="INSERT_OR_UPDATE">
        <application/>
        <key>Reason: The update set &lt;b&gt;{0}&lt;/b&gt; was closed by {1} on {2}</key>
        <language>en</language>
        <message>Reason: The update set &lt;b&gt;{0}&lt;/b&gt; was closed by {1} on {2}</message>
        <sys_class_name>sys_ui_message</sys_class_name>
        <sys_created_by>admin</sys_created_by>
        <sys_created_on>2020-09-28 20:17:02</sys_created_on>
        <sys_id>423f9b95dba39c1046b593c3e2961978</sys_id>
        <sys_mod_count>2</sys_mod_count>
        <sys_name>Reason: The update set &lt;b&gt;{0}&lt;/b&gt; was closed by {1} on {2}</sys_name>
        <sys_package display_value="Customer Outcomes Assignment" source="x_snc_customer_out">7d70ba10db879850bde16a3b13961924</sys_package>
        <sys_policy/>
        <sys_scope display_value="Customer Outcomes Assignment">7d70ba10db879850bde16a3b13961924</sys_scope>
        <sys_update_name>sys_ui_message_423f9b95dba39c1046b593c3e2961978</sys_update_name>
        <sys_updated_by>admin</sys_updated_by>
        <sys_updated_on>2020-12-29 16:11:49</sys_updated_on>
    </sys_ui_message>
</record_update>
